### DISCUSSION TITLE
Moving Map

### DISCUSSION THREAD

**Discussion by Warren-Whitfield** [pelicanmapping/osgearth#2517] [Support Q&A] [answered]
Hi
I would like to make the map move across the screen while my position depited by an icon or something stays stationary in the centre of the display.
I've tried m_pMouseEarthManip->setViewpoint(viewpoint), but this then interferes with the normal actions of dragging the map about the display. Is there another way to achieve this?
In my application it is own ship who should remain in the centre while the 'world' below moves as the ship is moving.
Any ideas will be appreciated.
Thanks

**Comment by gwaldron** [comment]
One way is to use tethering. If you use `Viewpoint::setNode()` your camera will track a node in the scene graph. You can then position something in the scene using a `GeoTransform` (for example) and point your Viewpoint at that.
```c++
auto target = new GeoTransform();
target->setPosition(...);
...
Viewpoint vp = earthManip->getViewpoint();
vp->setNode(target);
earthManip->setViewpoint(vp);
```

**Comment by Warren-Whitfield** [comment]
Ok, so that kinda works. By that I mean that ownship is now in fact stationary in the centre of the display and the world moves beneath it. However, I can't drag the map about the screen though. Is there a way that this could be made possible in this mode?

**Comment by gwaldron** [comment]
Sorry, I guess I don't understand what you want.

**Reply by Warren-Whitfield** [reply, marked as answer]
No worries, I'll make it work as is... :+1:
